### ISSUE TITLE
failure in sockaddr/test_uct_sockaddr.err_handle/4

### ISSUE THREAD

**Issue by brminich** [openucx/ucx#4357]
http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/12930/label=hpc-test-node-gpu,worker=1/consoleFull#-853622487b816df86-9afc-46eb-984f-1a9a125d73bb

```
3:30:14 [ RUN      ] sockaddr/test_uct_sockaddr.err_handle/4 <sockaddr/eth0>
23:30:14 [     INFO ] Testing fcfc:fcfc:209:44:225:90ff:fe94:755c:55240 Interface: eth0
23:30:14 [1572471014.146914] [hpc-test-node-gpu02:28241:0]   sockcm_iface.c:214  UCX  WARN  accept(fd=33) failed with non-recoverable error Bad file descriptor
23:30:14 [     INFO ] < failed to connect to fcfc:fcfc:209:44:225:90ff:fe94:755c:55240 >
23:30:14 /scrap/jenkins/workspace/hpc-ucx-pr/label/hpc-test-node-gpu/worker/1/contrib/../test/gtest/common/test.cc:273: Failure
23:30:14 Failed
23:30:14 Got 1 warnings during the test
23:30:14 
[  FAILED  ] sockaddr/test_uct_sockaddr.err_handle/4, where GetParam() = sockaddr/eth0 (3 ms)
```

**Comment by brminich** [comment]
should be fixed by #4352

**Comment by yosefe** [comment]
fixed by #4352
